src/ai/backend/manager/services/session/service.py

```diff
@@ -504,6 +507,8 @@ async def _commit_and_upload(
                 try:
                     cache_id = EventCacheDomain.BGTASK.cache_id(bgtask_id)
                     async for event in propagator.receive(cache_id):
+                        if isinstance(event, BgtaskUpdatedEvent):
+                            continue
                         if not isinstance(event, BaseBgtaskDoneEvent):
                             log.warning("unexpected event: {}", event)
```

Comment:
Since it semantically receives all BgtaskEvents, check for BgtaskEvent (in this case, if an event other than bgtask is input, it should be a warning log), and then skip Updated in the match case.